model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,48 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.client.active_instances
+    type: metric
+    metric_name: db.cosmosdb.client.active_instances
+    brief: "Number of active client instances"
+    instrument: histogram
+    unit: "{instance}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: attributes.db.cosmosdb.minimal
+    type: attribute_group
+    brief: 'Azure Cosmos DB attributes'
+    attributes:
+      - ref: db.system
```

Comment:
We don't  have a good way to show it properly using tooling yet. Currently when it's rendered it shows 50+ db.system values table and is confusing. 

We're doing something like 

```yaml
      # TODO: add db.system once https://github.com/open-telemetry/build-tools/issues/192 is possible
      # - ref: db.system
```

in other places and adding some text in markdown that system is populated to improve readability. Suggesting to do the same here.